ru/techdocs/api/client/hoodie.md

```diff
@@ -0,0 +1,302 @@
+  console.log(doneTodo.title, ' was done', t);
+});
+
+todoStore.findAll().done(function(allTodos) {
+  // take the first todo in list
+  // and mark it as done
+  markAsDone(allTodos[0]);
+});
+</code></pre>
+
+##### Notes
+> - Хорошая идея - придерживаться соглашений в наименовании, вида: 'object-type:what-happened' или 'what-happened'
+> - Начиная со второго параметра вы можете передать любые дополнительные данные для события.
+> - Если вы зарегистрировали хэндлер **hoodie.store('todo').on** и вызываете его через **hoodie.store.trigger**, предыдущий зарегистрированный хэндлер не вызывается.
+
+##### Примеры
+<pre><code>var todoStore = hoodie.store('todo');
+
+todoStore.on('trigger-test', function(num) {
+  // will only be called by the second trigger
+  console.log('triggered by', num);
+});
+
+hoodie.store.trigger('trigger-test', 'number 1');
+todoStore.trigger('trigger-test', 'number 2');
+hoodie.store(hoodie).trigger('trigger-test', 'number 3');
+</code></pre>
+
+
+<a id="request"></a>
+### request()
+
+**version:**      *> 0.2.0*
+
+*Отправляет запрос*
+
+<pre><code>hoodie.request(type, url, options);</code></pre>
+
+| option     | type     | description                                | required |
+| ---------- |:--------:|:------------------------------------------:|:--------:|
+| type       | string   | http verb, e.g. get, post, put or delete   | yes      |
+| url        | string   | relative path or absolute URL.             | yes      |
+| options    | object   | compare <a href="http://api.jquery.com/jquery.ajax/" target="_blank">http://api.jquery.com/jquery.ajax</a> | no       |
+
+
+##### Пример
+<pre><code>hoodie.request('http://example.com/something')
+  .done(renderSomething)
+  .fail(handleError);
+</code></pre>
+
+
+<a id="open"></a>
+### open()
+
+**version:**      *> 0.2.0*
+
+*Подключается к базе*
+
+<pre><code>hoodie.open('db-name');</code></pre>
+
+| option     | type     | description            | required |
+| ---------- |:--------:|:----------------------:|:--------:|
+| name       | string   | name of the database   | yes      |
+
+
+##### Пример
+<pre><code>var chat = hoodie.open('chatroom');
+chat.findAll('message')
+  .done(renderMessages)
+  .fail(handleError);
+</code></pre>
+
+
+<a id="checkConnection"></a>
+### checkConnection()
+
+**version:**      *> 0.2.0*
+
+*Проверяет доступность сервера*
+
+<pre><code>hoodie.checkConnection();</code></pre>
+
+##### Пример
+<pre><code>hoodie.checkConnection()
+  .done(renderGreenLight)
+  .fail(renderRedLight);
+</code></pre>
+
+
+<a id="isConnected"></a>
+### isConnected()
+
+**version:**      *> 0.2.0*
+
+*Возвращает true если бэкенд Hoodie доступен*
+
+<pre><code>hoodie.isConnected();</code></pre>
+
+##### Пример
+<pre><code>if (hoodie.isConnected()) {
+  alert('Looks like you are online!');
+} else {
+  alert('You are offline!');
+}
+</code></pre>
+
+
+<a id="extend"></a>
+### extend()
+
+**version:**      *> 0.2.0*
+
+*Расширяет hoodie API новой функциональностью*
```

Comment:
Words order in Russian is different, not

> hoodie API

but

> API Hoodie

Shouldn’t Hoodie start from capital letter?